.do/Makefile

```diff
@@ -88,6 +88,7 @@ update_dependencies:
 .PHONY: update_ingress_values
 update_ingress_values:
 	sed -i "s#<codacy-app.dns.internal>#${CODACY_URL}#g; s#<codacy-api.dns.internal>#${CODACY_URL}#g; s#http://##g" values.yaml
+	sed -i "s#<environment>#${NAMESPACE}#g" values.yaml
```

Comment:
This guarantees that each environment/namespace has its own properly named secret/certificate. This avoids issues with certmanager.